src/azure-cli-core/HISTORY.rst

```diff
@@ -3,6 +3,10 @@
 Release History
 ===============
 
+2.55.0
+++++++
+* Minor fixes
```

Comment:
Not sure what the logic that generates this `Minor fixes` is. We may want to look at the pipeline and adjust it to be more specific.